frontend/src/redux/reducers/finances.js

```diff
@@ -1,13 +1,6 @@
 const initialState = {
-  salaryBeforeTax: 0.0,
   salaryAfterTax: 87000,
-  shopping: [
-    {
-      id: 0,
-      description: "",
-      amount: 0.0,
-    },
-  ],
+  shoppingAmount: 0.0,
```

Comment:
just call it 'shopping' because we don't have any other shopping values and then we won't have to tack on 'amount' to all the other inputs that will be like this one